These are just helper functions, but if dynamic modules are disabled,
you get multiple definition errors in libformats.la.

--- origsrc/openbabel-2.2.1/src/formats/cifformat.cpp	2009-02-10 18:13:45.000000000 -0600
+++ src/openbabel-2.2.1/src/formats/cifformat.cpp	2009-06-22 12:31:45.330533700 -0500
@@ -964,7 +964,7 @@ namespace OpenBabel
   // Is this atom an oxygen in a water molecule
   // We know the oxygen is connected to one ion, but check for non-hydrogens
   // Returns: true if the atom is an oxygen and connected to two hydrogens and up to one other atom
-  bool isWaterOxygen(OBAtom *atom)
+  static bool isWaterOxygen(OBAtom *atom)
   {
     if (!atom->IsOxygen())
       return false;
--- origsrc/openbabel-2.2.1/src/formats/smilesformat.cpp	2009-02-10 17:06:41.000000000 -0600
+++ src/openbabel-2.2.1/src/formats/smilesformat.cpp	2009-06-22 12:31:24.718354700 -0500
@@ -2320,7 +2320,7 @@ namespace OpenBabel {
   // Is this atom an oxygen in a water molecule
   // We know the oxygen is connected to one ion, but check for non-hydrogens
   // Returns: true if the atom is an oxygen and connected to two hydrogens + one coordinated atom
-  bool isWaterOxygen(OBAtom *atom)
+  static bool isWaterOxygen(OBAtom *atom)
   {
     if (!atom->IsOxygen())
       return false;
